edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -77,6 +77,10 @@ var edx = edx || {};
 
             this.render();
         },
+        hideCourseNavBar: function() {
+            $(".wrapper-course-material").addClass('invisible');
```

Comment:
added the class `invisble` which will hides the visibility for the course navigation bar but the space for the navigation bar will be there.